docs/contributor-guide/developer-guide/system-design/data-intake.md

```diff
@@ -0,0 +1,47 @@
+# Data intake
+
+This page describes where various pieces of data related to user accounts originate.
+
+CiviForm's use of a flexible, admin-defined [backend data model](./backend-data-model.md) complicates investigations into where a given piece of data about an applicant comes from.
+
+## `ApplicantData`
+
+Generally speaking, data stored in [ApplicantData](https://github.com/civiform/civiform/blob/main/server/app/services/applicant/ApplicantData.java) (per the [backend data model](./backend-data-model.md) schema) originates from applicants (or TIs acting on an applicant's behalf) answering questions.
+
+### Hard-coded ApplicantData entries
+
+#### Well known paths
+
```

Comment:
It might be worth considering if we want to create support for Phone number data. While registeration, we ask for User's phone number. Where is this information stored? 
![image](https://github.com/civiform/docs/assets/101214724/91e39742-6f14-42ac-9bf1-8f178b1b08dc)

Reply:
There currently no special case for phone number data in the system as from the existing phone number type

Reply:
so we ask for phone number and not store it?

Reply:
No need to answer this @bion . It is more for our team to figure out where to go next.

Reply:
> so we ask for phone number and not store it?

This screenshot looks like it's from IDCS, so I imagine it's stored there. We could update our auth logic to copy it into civiform the same way we do with email.

Reply:
in that case, will phone number also be stored in Accounts table?

Reply:
I'm not sure, would need to think it through in a design